### PR TITLE
[move-cli] support saving and viewing events

### PR DESCRIPTION
**Pull request by sblackshear**
- Added `events` subdirectory of move_data/<addr> for storing emitted events. Subdirectories of `events` store event handles named by their "creation number". For example, if account address 0xA creates two handles and emits two events to each, the CLI will create two files `0xA/events/0` and `0xA/events/1` and store the two contract events emitted to each handle in the appropriate file.
- Added similar `move_data/<addr>/resources` and `move_data/<addr>/modules` subdirectories to partition the storage under each address more cleanly
- `move run` now stores emitted events on disk instead of discarding them
- `move view` now supports viewing events

### CODE REVIEW

File: language/tools/move-cli/tests/testsuite/module_run_view/args.exp [hunk 1]
```diff
@@ -6,10 +6,10 @@ Changed resource(s) under 2 address(es):
   Changed 1 resource(s) under address 0000000000000000000000000000000b:
     Added type 00000000::Test::R: [U64(7)]
 Discarding changes; re-run with --commit if you would like to keep them.
-Command `view move_data/0x0000000000000000000000000000000a/0x00000000000000000000000000000002::Test::R`:
-Resource not found.
-Command `view move_data/0x0000000000000000000000000000000b/0x00000000000000000000000000000002::Test::R`:
-Resource not found.
+Command `view move_data/0x0000000000000000000000000000000a/resources/0x00000000000000000000000000000002::Test::R`:
+Not found.
```
Comment (tzakian) [inline language/tools/move-cli/tests/testsuite/module_run_view/args.exp:10]: Possible regression? Shouldn't this say "Resource not found."?

**Review by tzakian — APPROVED** [review]
This look great! One possible issue, but happy for this to land as-is.

File: language/tools/move-cli/tests/testsuite/module_run_view/args.exp (hunk 1 above)
Comment (sblackshear) [reply language/tools/move-cli/tests/testsuite/module_run_view/args.exp:10]: Ah, good call. Not sure if that check is needed; let me try eliminating it.

**Pull request closed without merging**